README.md

```diff
@@ -15,10 +15,16 @@ Code licensed under the [MIT License](https://github.com/Microsoft/MSIX-PackageS
 ## Contribute
 This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
+Submit your own fixup(s) to the community:
+1. Create a private fork for yourself
+2. Make your changes in your private branch
+3. Create a Pull Request
+4. Target the main repo

```

Comment:
Maybe "3. Create a pull request into 'fork:Microsoft/MSIX-PackageSupportFramework' 'base:master'

and omit steps 4 and 5